### PR TITLE
feat: add community meeting_12.14

### PR DESCRIPTION
**Pull request by cqqqq777**
#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.
- [ ] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: meeting_notes/2023/CloudWeGo_community_meeting_12.14.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+1. 基础知识：介绍了反向代理与 WebSocket。
+2. 基本思路：通过 Hertz 提供 `http.Handler`，转发用户的 WebSocket 连接，最终实现代理服务器与真实服务器，用户与代理服务器的连接，代理服务器只负责消息转发。
+3. 具体实现：源码位于 https://github.com/hertz-contrib/reverseproxy，实现步骤如下：
+    1. 抽象 `WSReverseProxy` 主结构体，实现其构造方法 `NewWSReverseProxy`。
+    2. 实现核心方法 `ServeHTTP`，主要分为以下四步：
+        1. 准备请求头
+        2. 代理服务器与后端服务器建立 WebSocket 连接
+        3. 客户端与代理服务器建立 WebSocket 连接
+        4. 连接通信
+4. 示例代码：于 https://github.com/hertz-contrib/reverseproxy 提供了使用示例代码，并简单介绍。
```
Comment (justlorain) [inline meeting_notes/2023/CloudWeGo_community_meeting_12.14.md:29]: 示例代码在：https://github.com/cloudwego/hertz-examples/tree/main/reverseproxy/websocket

Comment (cqqqq777) [reply meeting_notes/2023/CloudWeGo_community_meeting_12.14.md:29]: 收到

File: meeting_notes/2023/CloudWeGo_community_meeting_12.14.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### 议程三：Kitex [obs](https://github.com/kitex-contrib/obs-opentelemetry) slog 扩展介绍 @张鑫
+
+1. slog 简介：slog 是 Go 1.21 版本新增的结构化日志库，核心为 `Logger` 结构体和 `Handler` 接口。用户可以通过实现 `Handler` 接口实现自定义日志操作。
+2. obs slog 扩展思路：
+    1. 实现 Kitex `klog.FullLogger` 接口，使得 slog 适配 klog
+    2. 自定义实现 `traceHandler`
+    3. 在 `JSONHandler` 的基础上扩展 `Handle`方法，处理链路追踪的日志记录
+3. Slog [More info](https://link.zhihu.com/?target=https%3A//go.googlesource.com/proposal/%2B/master/design/56345-structured-logging.md)
```
Comment (XZ0730) [inline meeting_notes/2023/CloudWeGo_community_meeting_12.14.md:38]: 链接可以改成 https://go.googlesource.com/proposal/+/master/design/56345-structured-logging.md

**Review by XZ0730 — COMMENTED** [review]
LGTM

File: meeting_notes/2023/CloudWeGo_community_meeting_12.14.md (hunk 2 above)
Comment (cqqqq777) [reply meeting_notes/2023/CloudWeGo_community_meeting_12.14.md]: 收到

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**